ansible/manila-client.yml

```diff
@@ -9,6 +9,7 @@
       vars:
         os_manila_mount_action: "lookup"
         os_manila_mount_share_name: "{{ ceph_mount_share_name }}"
+        os_manila_mount_share_user: "{{ ceph_mount_share_user }}"
```

Comment:
I don't see the ceph_mount_share_user variable defined anywhere. OTOH, os_manila_mount_share_user is now defined in group_vars, so can we just use that and remove this line?

Reply:
Can we also then remove `os_manila_mount_share_name: "{{ ceph_mount_share_name }}"` which we can define under group vars?